### PR TITLE
Make New Service Schedule work in Experience Cloud

### PR DESCRIPTION
**Pull request by bethbrains**
# Critical Changes

# Changes
- The custom wizard override for creating a new Service Schedule now works in Experience Cloud.

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage  
- ~~CRUD/FLS is enforced in Apex~~
- ~~Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
- ~~Field sets are updated to account for new fields~~
- [x] UX approval or UX not necessary
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [x] PR contains draft release notes
- [ ] QE story level testing completed

### CODE REVIEW

File: force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.html [hunk 1]
```diff
@@ -131,7 +131,7 @@
                                         {labels.every}
                                     </lightning-layout-item>
                                     <lightning-layout-item
-                                        size="3"
+                                        size="4"
```
Comment (bethbrains) [inline force-app/main/default/lwc/newServiceSchedule/newServiceSchedule.html:134]: NOTE: The width of the modal is much narrower in experience cloud, so I made this slightly bigger so the field is usable.

File: force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html [hunk 2]
```diff
@@ -8,139 +8,140 @@
...
-            <template if:true={isStep4}>
-                <c-service-schedule-review
-                    service-schedule-model={serviceScheduleModel}
-                    onloaded={handleLoaded}
-                    participant-columns={participantColumns}
-                ></c-service-schedule-review>
-            </template>
-            <c-modal header={labels.goBack} is-nested="true">
-                {labels.backWarning}
+    <!-- fake header for Experience Cloud where the component brings its own modal -->
```
Comment (bethbrains) [inline force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html:11]: Note: This doesn't appear exactly the same way and doesn't float.

**Review by bmuniswamy — COMMENTED** [review]
Just one question Beth. This looks good

File: force-app/main/default/aura/createServiceSchedule/createServiceScheduleController.js [hunk 3]
```diff
@@ -9,6 +9,11 @@
 
 ({
     doInit: function(component, event, helper) {
+        if ($A.get("$Site")) {
```
[the remark this answers is not in the document]
Comment (bethbrains) [reply force-app/main/default/aura/createServiceSchedule/createServiceScheduleController.js:12]: Fair point!! I guess i was feeling verbose. Haha. I'll change it. :)

File: force-app/main/default/aura/createServiceSchedule/createServiceScheduleController.js [hunk 4]
```diff
@@ -9,6 +9,7 @@
 
 ({
     doInit: function(component, event, helper) {
+        component.set("v.isCommunity", $A.get("$Site") ? true : false);
```
Comment (bethbrains) [inline force-app/main/default/aura/createServiceSchedule/createServiceScheduleController.js:12]: must cast as a boolean. can't pass truthy. :(

File: force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.js [hunk 5]
```diff
@@ -159,6 +160,12 @@ export default class ServiceScheduleCreator extends NavigationMixin(LightningEle
         return this.currentStep.value === 3;
     }
 
+    get warningModal() {
+        return this.isCommunity
```
Comment (jjbennett) [inline force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.js:164]: Ahh community cloud why lol, nice fix!

File: force-app/main/default/lwc/serviceScheduleCreatorWrapper/__tests__/serviceScheduleCreatorWrapper.test.js [hunk 6]
```diff
@@ -0,0 +1,63 @@
+/*
+ *
+ *  * Copyright (c) 2020, salesforce.com, inc.
```
Comment (jjbennett) [inline force-app/main/default/lwc/serviceScheduleCreatorWrapper/__tests__/serviceScheduleCreatorWrapper.test.js:3]: It's hard to believe but it is 2021 now :)

**Review by jjbennett — APPROVED** [review]
Great work just those few copyright years to fix up. Thanks!

**Review by jjbennett — APPROVED** [review]

**Pull request merged**